File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] a = new int [nums.length * 2];
        int n = nums.length;
        int i = 0;

        while (i < n) {
            a[i] = nums[i];
            i++;
        }
        int j = 0;
        while (j < n) {
            a[i] = nums[j];
            i++;
            j++;
        }
        return a;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        e = 0
        currE = 0
        for i in nums:
            if i == 1:
                currE += 1
            else:
                currE = 0
            e = max(e, currE)
        return e
            